LICENSE-binary

```diff
@@ -498,6 +498,7 @@ org.slf4j:slf4j-reload4j:1.7.36
 CDDL 1.1 + GPLv2 with classpath exception
 -----------------------------------------
 
+com.github.jnr:jnr-posix:3.1.19
```

Comment:
actually it's not CDDL 1.1 + GPLv2 with classpath exception. Eclipse Public License 1/.02.0 is acceptable though.

For more details, check out Apache's thirdparty license policy: https://www.apache.org/legal/resolved.html

Reply:
Done.You are right, I added EPL 2.0 section.